### PR TITLE
chore: use gw-codegen to generate the go types

### PR DESCRIPTION
**Pull request by matt-boris**

### CODE REVIEW

**Comment by petemoore** [comment]
Nice one @matt-boris !

File: .taskcluster.yml [hunk 1]
```diff
@@ -43,6 +43,7 @@ tasks:
                 git -c advice.detachedHead=false checkout '${head_rev}'
 
                 go install github.com/taskcluster/taskcluster/v51/tools/jsonschema2go/jsonschema2go@latest
+                go install github.com/taskcluster/taskcluster/v51/workers/generic-worker/gw-codegen@latest
```
Comment (petemoore) [inline .taskcluster.yml:46]: you can delete line 45 here, line 46 can replace it

Comment (matt-boris) [reply .taskcluster.yml:46]: Still need `jsonschema2go` for the go generate of the `generated_types_test.go` file :)

Comment (petemoore) [reply .taskcluster.yml:46]: my mistake!

**Pull request merged**